src/Util/HTTPSignature.php

```diff
@@ -459,7 +459,14 @@ public static function getSigner($content, $http_headers)
 			}
 		}
 
-		/// @todo Check if the signed date field is in an acceptable range
+		//  Check if the signed date field is in an acceptable range
+		if (in_array('date', $sig_block['headers'])) {
+			$diff = abs(strtotime($headers['date']) - time());
+			if ($diff > 300) {
```

Comment:
Where is this value coming from? Is it a RFC or wet finger kind of number?

Reply:
Some are using 30 seconds, some are using a full day. So I thought that 300 seconds are nice as well. Should be enough for bad configured servers without ntp and too short for replay attacks.